### ISSUE TITLE
Doc:  Clarify where input settings should be placed when using filebeat.config.inputs

### ISSUE THREAD

**Issue by ppf2** [elastic/beats#19148]
6.8.9

When using [external config](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-configuration-reloading.html#load-input-config) files, it is unclear what input settings are actually allowed under the `filebeat.config.inputs` construct.  For example, if the user configures clean_inactive under `filebeat.config.inputs`, it does not throw a configuration error and treats it as a valid config, but it doesn't appear to apply to the inputs that are configured within the external files.

```
filebeat.config.inputs:
  enabled: true
  path: prospectors/*.yml
  clean_inactive: 24h
```

Specifying these settings within the external configuration files work.  While the example on this page does show that scan_frequency is configured as part of the input block, because we are accepting these settings at the `filebeat.config.inputs` level, it will be helpful to clarify that these settings must all go into the external files, etc..

**Comment by ycombinator** [comment]
I looked through the Filebeat code, specifically this path:
1. https://github.com/elastic/beats/blob/93a4591cf432819473cd69c6dee5f267d48f936b/filebeat/beater/filebeat.go#L292
2. https://github.com/elastic/beats/blob/93a4591cf432819473cd69c6dee5f267d48f936b/filebeat/beater/crawler.go#L81-L87. 
3. https://github.com/elastic/beats/blob/93a4591cf432819473cd69c6dee5f267d48f936b/libbeat/cfgfile/reload.go#L107-L123
4. https://github.com/elastic/beats/blob/93a4591cf432819473cd69c6dee5f267d48f936b/libbeat/cfgfile/reload.go#L38-L44
5. https://github.com/elastic/beats/blob/93a4591cf432819473cd69c6dee5f267d48f936b/libbeat/cfgfile/reload.go#L58-L64
6. https://github.com/elastic/beats/blob/93a4591cf432819473cd69c6dee5f267d48f936b/libbeat/cfgfile/reload.go#L125-L160

According to the code, we are not expecting any input settings to be specified directly under `filebeat.config.inputs`. They must be specified under each input defined in the external input configuration files. +1 to clarifying this in the documentation.

**Comment by ppf2** [comment]
Thanks for confirming!  Submitted a [PR](https://github.com/elastic/beats/pull/19284) to clarify documentation until we start [validating configuration settings](https://github.com/elastic/beats/issues/4416) for Beats.

**Comment by dedemorton** [comment]
Closed by #19284
